File: frontend/src/app/dto/attendance/attendance.update.res.dto.ts
import { AttendanceUpdateDataResDto } from "./attendance.update.data.res.dto";

export class AttendanceUpdateResDto{
    private _data! : AttendanceUpdateDataResDto
	private _message! : string

    public get data(): AttendanceUpdateDataResDto {
        return this._data;
    }

    public set data(data: AttendanceUpdateDataResDto) {
        this._data = data;
    }

    public get message(): string {
        return this._message;
    }

    public set message(message: string) {
        this._message = message;
    }
    
}

File: backend/src/main/java/com/lawencon/elearn/dto/attendance/AttendanceUpdateResDto.java
package com.lawencon.elearn.dto.attendance;

public class AttendanceUpdateResDto {
	private AttendanceUpdateDataResDto data;
	private String message;
	public AttendanceUpdateDataResDto getData() {
		return data;
	}
	public void setData(AttendanceUpdateDataResDto data) {
		this.data = data;
	}
	public String getMessage() {
		return message;
	}
	public void setMessage(String message) {
		this.message = message;
	}
}
